pages/bitmark-references/bitmark-node-software/README.md

```diff
@@ -2,65 +2,70 @@
 # Bitmark Node Software
 
 
-## [Bitmark Blockchain Overview](bitmark-blockchain-overview.md)
+## [Bitmark Blockchain Technical Overview](bitmark-blockchain-overview.md)
 
 * [Definitions](bitmark-blockchain-overview.md#definitions)
 * [Transactions: Asset, Issue and Transfer Records](bitmark-blockchain-overview.md#transactions-asset-issue-and-transfer-records)
     * [Asset Record](bitmark-blockchain-overview.md#asset-record)
     * [Issue Record](bitmark-blockchain-overview.md#issue-record)
     * [Transfer Record](bitmark-blockchain-overview.md#transfer-record)
-  * [Transactions: Share](bitmark-blockchain-overview.md#transactions-share)
-    * [Bitmark Share](bitmark-blockchain-overview.md#bitmark-share)
-    * [Share Grant](bitmark-blockchain-overview.md#share-grant)
-    * [Share Swap](bitmark-blockchain-overview.md#share-swap)
-  * [Transactions: Block ownership record](bitmark-blockchain-overview.mdd#transactions-block-ownership-record)
-    * [Block Foundation](bitmark-blockchain-overview.mdd#block-foundation)
-    * [Block Owner Transfer](bitmark-blockchain-overview.md#block-owner-transfer)
-  * [Blockchain structure](bitmark-blockchain-overview.md#blockchain-structure)
-    * [Block Header](bitmark-blockchain-overview.md#block-header)
-    * [Merkle tree](bitmark-blockchain-overview.md#merkle-tree)
-    * [Block Hashing (argon2, difficulty, proof of work \.\.\.)](bitmark-blockchain-overview.md#block-hashing-argon2-difficulty-proof-of-work-)
-
-
-## [Block Verification and Synchronization](block-verification-and-synchronization.md#block-verification-and-synchronization)
-
-* [Bitmark Block](block-verification-and-synchronization.md#bitmark-block)
-* [Different types of transactions](block-verification-and-synchronization.md#different-types-of-transactions)
+* [Transactions: Share](bitmark-blockchain-overview.md#transactions-share)
+  * [Share Balance Record](bitmark-blockchain-overview.md#share-balance-record)
+  * [Share Grant Record](bitmark-blockchain-overview.md#share-grant-record)
+  * [Share Swap Record](bitmark-blockchain-overview.md#share-swap-record)
+* [Transactions: Block ownership record](bitmark-blockchain-overview.md#transactions-block-ownership-record)
+  * [Block Foundation Record](bitmark-blockchain-overview.md#block-foundation-record)
+  * [Block Owner Transfer Record](bitmark-blockchain-overview.md#block-owner-transfer-record)
+* [Blockchain Structure](bitmark-blockchain-overview.md#blockchain-structure)
+  * [Block Header](bitmark-blockchain-overview.md#block-header)
+  * [Merkle Tree](bitmark-blockchain-overview.md#merkle-tree)
+  * [Block Hashing (argon2, difficulty, proof of work \.\.\.)](bitmark-blockchain-overview.md#block-hashing-argon2-difficulty-proof-of-work-)
+  
+## [Block Validation and Synchronization](block-verification-and-synchronization.md#block-verification-and-synchronization)
```

Comment:
It seems you use "Validation"  instead of "Verification" here but in other parts you keep Verification... any particular reason?